### PR TITLE
docs: add network events documentation to mkdocs

### PR DESCRIPTION
**Pull request by rafaeldtinoco**
Execute:

```
make -f builder/Makefile.mkdocs mkdocs-serve
```

To read and test it.

### CODE REVIEW

File: docs/docs/tracing/network-events.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+    ```
+
+## Network Based Signatures
+
+It is possible to create Golang (or Rego) signatures for the network events. If you haven't read about how to create signatures, do it [HERE](../../detecting/golang/).
+
+!!! Examples Note
+    Bellow is an example of how to create a signature for the `net_packet_dns` event. This same example is used by Tracee CI/CD tests and can be found at the [GitHub repository](https://github.com/aquasecurity/tracee/tree/main/tests/e2e-net-rules), together with some other signatures for the network events.
```
Comment (geyslan) [inline docs/docs/tracing/network-events.md:218]: I think this is broken on my local preview:

image

File: docs/docs/tracing/network-events.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    }
+
+    return nil
+}
+
+func (sig *e2eDNS) OnSignal(s detect.Signal) error {
+    return nil
+}
+
+func (sig *e2eDNS) Close() {}
```
Comment (geyslan) [inline docs/docs/tracing/network-events.md:342]: Should do nothing?

File: docs/docs/tracing/network-events.md (hunk 1 above)
Comment (rafaeldtinoco) [reply docs/docs/tracing/network-events.md:218]: This type of stuff is rendered only in mkdocs (not in github markdown).

File: docs/docs/tracing/network-events.md (hunk 2 above)
Comment (rafaeldtinoco) [reply docs/docs/tracing/network-events.md:342]: Yep, only an example (and a test).

File: docs/docs/tracing/network-events.md (hunk 1 above)
Comment (rafaeldtinoco) [reply docs/docs/tracing/network-events.md:218]: image

**Review by geyslan — APPROVED** [review]
LGTM

**Pull request merged**